BoneLib/BoneLib/HelperMethods.cs

```diff
@@ -66,6 +66,101 @@ public static void SpawnCrate(SpawnableCrateReference crateReference, Vector3 po
+        public static void LoadLevel(string barcode)
+        {
+            LoadLevel(barcode, CommonBarcodes.Maps.LoadDefault);
+        }
+
+        /// <summary>
+        /// Loads a level from a crate reference
+        /// </summary>
+        /// <param name="level">The crate reference to load</param>
+        /// <param name="loadLevel">The crate reference for the loading scene</param>
+        public static void LoadLevel(LevelCrateReference level, LevelCrateReference loadLevel)
+        {
+            LoadLevel(level.Barcode.ID, loadLevel.Barcode.ID);
+        }
+
+        /// <summary>
+        /// Loads a level from a barcode
+        /// </summary>
+        /// <param name="levelBarcode">The barcode of the level</param>
+        /// <param name="loadLevelBarcode">The barcode of the loading scene</param>
+        public static void LoadLevel(string levelBarcode, string loadLevelBarcode)
+        {
+            SceneStreamer.Load(new Barcode(levelBarcode), new Barcode(loadLevelBarcode));
+        }
+
+        /// <summary>
+        /// Loads a level from a crate reference with load fade
+        /// </summary>
+        /// <param name="level">The crate reference to load</param>
+        /// <param name="fadeFast">When true, loads with a faster fade</param>
+        public static void FadeLoadLevel(LevelCrateReference level, bool fastFade = false)
+        {
+            FadeLoadLevel(level.Barcode.ID, CommonBarcodes.Maps.LoadDefault, fastFade);
+        }
+
+        /// <summary>
+        /// Loads a level from a barcode with load fade
+        /// </summary>
+        /// <param name="barcode">The barcode of the level</param>
+        /// <param name="fadeFast">When true, loads with a faster fade</param>
+        public static void FadeLoadLevel(string barcode, bool fastFade = false)
+        {
+            FadeLoadLevel(barcode, CommonBarcodes.Maps.LoadDefault, fastFade);
+        }
+
+        /// <summary>
+        /// Loads a level from a crate reference with load fade
+        /// </summary>
+        /// <param name="level">The crate reference to load</param>
+        /// <param name="loadLevel">The crate reference for the loading scene</param>
+        /// <param name="fadeFast">When true, loads with a faster fade</param>
+        public static void FadeLoadLevel(LevelCrateReference level, LevelCrateReference loadLevel, bool fastFade = false)
+        {
+            FadeLoadLevel(level.Barcode.ID, loadLevel.Barcode.ID, fastFade);
+        }
+
+        /// <summary>
+        /// Loads a level from a barcode with load fade
+        /// </summary>
+        /// <param name="levelBarcode">The barcode of the level</param>
+        /// <param name="loadLevelBarcode">The barcode of the loading scene</param>
+        /// <param name="fadeFast">When true, loads with a faster fade</param>
+        public static void FadeLoadLevel(string levelBarcode, string loadLevelBarcode, bool fastFade = false)
```

Comment:
Documentation for LoadLevel & FadeLoadLevel with fastFade params should be corrected. Contains for param with name "fadeFast" which does not exist and instead is fastFade.